packages/sdk/docker-bake.hcl

```diff
@@ -5,7 +5,7 @@ target "default" {
   inherits = ["docker-metadata-action", "docker-platforms"]
   args = {
     BASE_IMAGE               = "debian:bookworm-20230814"
-    MACHINE_EMULATOR_VERSION = "0.15.0"
+    MACHINE_EMULATOR_VERSION = "0.15.2"
```

Comment:
the changeset message mentions 0.16.2 and here we use 0.15.2, is that right?

Reply:
Yes, it's right, emulator version is 0.15.2 and sdk version is 0.16.2